newIDE/app/src/EventsSheet/InlineParameterEditor.js

```diff
@@ -123,7 +123,9 @@ export default class InlineParameterEditor extends React.Component<
           parameterIndex={this.props.parameterIndex}
           value={instruction.getParameter(this.props.parameterIndex)}
           onChange={this.props.onChange}
-          onRequestClose={this.props.onRequestClose}
```

Comment:
This change seems useless?

Reply:
Ah this is because of the fact that you're using event to test what to do. As explained in other comments, let's try to find a more explicit approach ;)